### PR TITLE
wazo-confd: add /status endpoint

### PR DESCRIPTION
**Pull request by nissou91**
Depends-On: https://github.com/wazo-platform/wazo-confd-client/pull/33

### CODE REVIEW

**Comment by nissou91** [comment]
recheck

**Comment by nissou91** [comment]
recheck

**Comment by sduthil** [comment]
recheck

Just running another one, to confirm that the test is not flaky.

File: integration_tests/suite/status/test_confd_status.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
```
Comment (pc-m) [inline integration_tests/suite/status/test_confd_status.py:1]: Can you move this file to `integration_tests/suite/base/test_status.py`

**Review by pc-m — CHANGES_REQUESTED** [review]

File: integration_tests/suite/status/test_confd_status.py (hunk 1 above)
Comment (nissou91) [reply integration_tests/suite/status/test_confd_status.py]: Done :)

**Comment by nissou91** [comment]
recheck

**Review by pc-m — APPROVED** [review]

**Comment by nissou91** [comment]
recheck

**Review by sduthil — APPROVED** [review]

**Pull request merged**